### PR TITLE
Add bamsplit helper function

### PR DESCRIPTION
**Pull request by jvrakor**
@dblenkus

### CODE REVIEW

File: resdk/analysis/chip_seq.py [hunk 1]
```diff
@@ -17,6 +17,48 @@ def is_background(sample):
...
+
+    results = []
+
+    if not isinstance(resource, list):
+        resource = [resource]
+
+    for single_resource in resource:
+        for sample in get_samples(single_resource):
+            bam = sample.get_bam()
+            inputs['bam'] = bam.id
```
Comment (dblenkus) [inline resdk/analysis/chip_seq.py]: Doesn't this work without `.id`? It think you can just write: `inputs['bam'] = sample.get_bam()`.

Comment (jvrakor) [reply resdk/analysis/chip_seq.py]: It doesn't work without `.id`. But I can rewrite it as  `inputs['bam'] = sample.get_bam().id`

Comment (dblenkus) [reply resdk/analysis/chip_seq.py]: It should work without `.id`. Please check why it doesn't. https://github.com/genialis/resolwe-bio-py/blob/master/resdk/resolwe.py#L292

Comment (mstajdohar) [reply resdk/analysis/chip_seq.py]: @jvrakor Would you print `type(sample.get_bam()).__name__`? The `get_data_id` function checks if `type(data).__name__ == 'Data'`, so my hypothesis is this condition is not satisfied.

**Pull request merged**